modules/custom/az_migration/config/install/az_migration.settings.yml

```diff
@@ -0,0 +1,5 @@
+migrate_d7_protocol: "https"
+migrate_d7_filebasepath: "dev-shantzios-d7.pantheonsite.io/"
```

Comment:
Can this be generic?

Reply:
This default value, We can update using the drush command as mentioned in readme file. Thanks